internal/core/software_renderer.rs

```diff
@@ -145,6 +150,32 @@ impl SoftwareRenderer {
         }
     }
 
+    /// Create a new Renderer for a given window.
+    ///
+    /// The `repaint_buffer_type` parameter specify what kind of buffer are passed to [`Self::render`]
+    ///
+    /// The `window` parameter can be coming from [`Rc::new_cyclic()`](alloc::rc::Rc::new_cyclic())
+    /// since the `WindowAdapter` most likely own the Renderer
+    #[doc(hidden)]
+    pub fn new_without_window(repaint_buffer_type: RepaintBufferType) -> Self {
```

Comment:
```suggestion
    pub(crate) fn new_without_window(repaint_buffer_type: RepaintBufferType) -> Self {
```

Should it maybe be new_with_repaint_buffer_type?
But new_without_window is fine as well.